docs/migrations/web/MIGRATION-v3.md

```diff
@@ -278,6 +278,17 @@ The Toast component now has its own close button and link.
 
    - `<button class="Button Button--square Button--inverted">` → `<button class="ToastBar__close">`
 
+### Link `hasVisitedStyleAllowed`
+
+The `link-allow-visited` class was added. This class allows the link to have visited state style. There is also a change in the use of the visited state style for the `typography-heading` class, which had previously this state set by default.
```

Comment:
typography-heading class does not exist. The selector was partial, so it matches any heading typography helper. We should reword it somehow.

```suggestion
The `link-allow-visited` class was added. This class allows the link to have visited state style. There is also a change in the use of the visited state style for the typography heading classes, which had previously this state set by default.
```

suggestion?

Reply:
LGTM. Reworded